stdlib/public/core/MemoryLayout.swift

```diff
@@ -44,9 +41,29 @@ public enum MemoryLayout<T> {
 }
 
 extension MemoryLayout {
+  /// Returns the contiguous memory footprint of `T`.
+  ///
+  /// Does not include any dynamically-allocated or "remote" storage. In
+  /// particular, `MemoryLayout(ofValue: x).size`, when `x` is a class instance,
```

Comment:
Typo in documentation comment: `MemoryLayout(ofValue: x).size` => `MemoryLayout.size(ofValue: x)`